api/core/v1beta2/machine_phase_types.go

```diff
@@ -45,6 +45,10 @@ const (
 	// become a Kubernetes Node in a Ready state.
 	MachinePhaseRunning = MachinePhase("Running")
 
```

Comment:
@fabriziopandini Let's discuss conversion via Zoom, it's not super straightforward